lib/registerSchema.ts

```diff
@@ -0,0 +1,38 @@
+import * as z from 'zod';
+
+import { ErrorMessages } from '@/constants/categories';
+
+export const RegisterSchema = z.object({
+  mbti: z.string().min(1, { message: ErrorMessages.mbti }),
+  name: z.string().min(1, { message: ErrorMessages.name }),
+  gender: z.string().min(1, { message: ErrorMessages.gender }),
+  email: z.string().email(ErrorMessages.email),
+  instagramId: z.string(),
+  age: z.coerce.number().refine(
+    (val) => {
+      // FIXME: 값을 입력하지 않았는데도 undefined -> 0으로 됨
+      if (val !== undefined && val !== 0) {
+        return val >= 20 && val <= 100;
+      }
```

Comment:
```suggestion
  age: z.coerce.number().int().gte(20).lte(100)
```

(p4) 이 방식으로 하면 문제가 있을까요?